app/models/user.rb

```diff
@@ -2,6 +2,8 @@ class User < ApplicationRecord
 
   has_secure_password
 
+  has_many :counselors
```

Comment:
A user doesn't have many counselors. If anything it is a one-to-one, but definitely not a one-to-many!